bin/bb8/remote_file_manager.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import docker
+from shellescape import quote
+
+from .remote_paths import RemotePaths
+
+
+class RemoteFileManager(object):
+    def __init__(self, paths: RemotePaths, docker_client=docker.from_env()):
+        self.paths = paths
+        self.docker_client = docker_client
+
+    def run_remote_cmd(self, remote_cmd):
+        cmd = ["ssh", self.paths.host, remote_cmd]
+        volumes = {
+            "bb8_ssh": {"bind": "/root/.ssh", "mode": "ro"}
+        }
+        return self.docker_client.containers.run("instrumentisto/rsync-ssh",
+                                                 command=cmd,
+                                                 volumes=volumes,
+                                                 remove=True)
+
+    def get_metadata(self):
+        remote_cmd = 'cat {path} || :'.format(path=self.paths.metadata_file)
```

Comment:
Can you explain this?  It looks like this is `:` which I am informed is `true` but with compatiblity with really ancient shells?  Can we use `true` instead?  It'll be there on alpine and easier to read in future.

So my interpretation here is "cat this file if it exists otherwise exit quietly?"

Reply:
Yup, that's correct. `:` was suggested when I googled. Now change to `true`. I didn't want to get an exit code when that file doesn't exist, as it would be too hard to distinguish from errors at the SSH or docker levels.

Reply:
I'd never seen `:` before! I think your approach is :100: the right thing to do returning 0